lib/commons/builder/queries/information_from_positions.rq.liquid

```diff
@@ -0,0 +1,33 @@
+SELECT DISTINCT
+  ?position {% lang_select 'position_name' %}
+  ?positionType
+  ?adminAreaTypes
+  ?adminArea {% lang_select 'admin_area' %}
+  ?positionSuperclass {% lang_select 'position_superclass' %}
+  ?legislature {% lang_select 'legislature' %}
+WHERE {
+  {
+    {% include 'select_admin_areas_for_country' %}
+  }
+  {
+    VALUES ?position { {{ positions_values }} }
+    {% lang_options 'position_name' '?position' %}
+    ?legislature wdt:P527 ?position .
```

Comment:
I think this needs a "has parts of class" alternative too, as that's e.g. how the UK cabinet is related to the Prime Minister class.
Furthermore, Scotland's cabinet is modelled per-government, which means it has lots of non-extant cabinets we need to ignore.
Put together, something like this:
```
    ?legislature wdt:P527|wdt:P2670 ?position .
    MINUS { ?legislature wdt:P576|wdt:P582 ?legislatureEnd }
```